pkg/gardenlet/controller/backupentry/backup_entry.go

```diff
@@ -65,13 +69,15 @@ func NewBackupEntryController(ctx context.Context, clientMap clientmap.ClientMap
 	}
 
 	controller := &Controller{
-		gardenClient:        gardenClient.Client(),
-		config:              config,
-		reconciler:          newReconciler(clientMap, recorder, config),
-		recorder:            recorder,
-		backupEntryInformer: backupEntryInformer,
-		backupEntryQueue:    workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "BackupEntry"),
-		workerCh:            make(chan int),
+		gardenClient:              gardenClient.Client(),
+		config:                    config,
+		reconciler:                newReconciler(clientMap, recorder, logger.Logger, config),
+		migrationReconciler:       newMigrationReconciler(clientMap, logger.Logger, config),
```

Comment:
Side-note for the next reconciler that will be added: `logger.Logger` is deprecated. Please use `logr` for new code. You can find instructions in https://github.com/gardener/gardener/pull/5175.

Reply:
I am aware about the logging change, but since all other controllers in `gardenlet` are still using the old logger, it's rather hard to figure out what to do (and the logging guideline was not available until very recently). Let's please keep it as it is for now and clean it up later, together will all existing controllers.